lib/registration.pm

```diff
@@ -153,10 +168,23 @@ sub yast_scc_registration {
             $timeout = 900;    # installation of an addon may take long
         }
         else {
-            assert_screen 'yast_scc-installation-summary', 900;
-            send_key "alt-f";
+            # yast may pop up a reboot prompt window after installation
```

Comment:
When does YaST pop up a reboot window? We should be asserting that YaST asks to reboot when it should, and asserting that YaST doesn't ask to reboot when it shouldn't